src/helpers/getDiscoverWeekly.js

```diff
@@ -1,10 +1,25 @@
-'use strict';
 
 const request = require('request');
 
 module.exports = (req, reply) => {
   return (err, response, body) => {
-    console.log(body);
-    reply('<h1>IM WORKING</h1>');
+    const headerObj = Object.assign({}, response);
```

Comment:
I don't think we actually need this step in the end. The response object is only passed to the callback when it has resolved.